src/window/city.c

```diff
@@ -63,13 +63,27 @@ static void draw_paused_and_time_left(void)
     }
 }
 
+static void draw_cancel_construction(void)
+{
+    if (!mouse_get()->is_touch || !building_construction_type()) {
+        return;
+    }
+    uint8_t cancel_button_text[] = { 'X', 0 };
```

Comment:
Maybe simplified?
```suggestion
    uint8_t* cancel_button_text = "X";
```

Reply:
You're looking at an outdated commit, beware